packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -2138,6 +2138,30 @@ export class ProjectModel {
         return { name: translatedToExplore.name };
     }
 
+    async deleteVirtualView(projectUuid: string, name: string) {
```

Comment:
I think we should start considering moving all the `virtual code` into its own `controller + service + model` . We can make a `chore-lightsmash` ticket for it, what do you think ?

Reply:
ooh I like that @rephus 
And maybe a new `features`/ folder for this in the FE!